www/test/functional/page-objects/report.js

```diff
@@ -21,19 +21,27 @@
   };
 
   this.fillRisk = function (risk) {
-    return element(by.cssContainingText('option', risk)).click();
+    element(by.cssContainingText('option', risk)).click();
+    return this;
   };
 
   this.submitButtonClick = function() {
-    return element(by.css('[ng-click="submit(reportForm.$valid)"]')).click();
+    element(by.css('[ng-click="submit(reportForm.$valid)"]')).click();
+    return this;
```

Comment:
I just changed this return to allow call this method in a nested way from the test.